validation/oauth_config.xsd

```diff
@@ -25,6 +25,7 @@
   <xs:complexType name="pluginOAuthConfigType">
     <xs:sequence>
       <xs:element type="xs:string" name="dbclass"/>
+      <xs:element type="xs:string" name="oauthConfigId" maxOccurs="1" minOccurs="0"/>
```

Comment:
Ensure there is test coverage in the connector-packager XSD validation. You may be able to augment one of the existing [oauth tests](https://github.com/tableau/connector-plugin-sdk/blob/master/connector-packager/tests/test_xsd_validator.py#L124) and its [files](https://github.com/tableau/connector-plugin-sdk/blob/master/connector-packager/tests/test_resources/oauth_connector/oauth-config.xml), or make a new test, depending on the plan for the one below.